internal/bminventory/inventory.go

```diff
@@ -4492,6 +4492,19 @@ func (b *bareMetalInventory) handlerClusterInfoOnRegisterInfraEnv(
 	return nil
 }
 
+func (b *bareMetalInventory) getActualImageType(cpuArchitecture string, imageType models.ImageType) models.ImageType {
+	if imageType == "" {
+		return ""
+	}
+
+	if cpuArchitecture == models.ClusterCPUArchitectureS390x && imageType != models.ImageTypeMinimalIso {
```

Comment:
it should be !=  models.ImageTypeFullIso

Reply:
I added a comment above the condition so it will be more clear